### PR TITLE
Hotfix: Clean code and use 2 indent

### PR DESCRIPTION
**Pull request by darmawan01**
# Description

Clean unused code/import, also replace the indent to 2

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

-

**Test Configuration**:
* OS:
* Toolchain:

# Checklist:

- [x] Choose at least one reviewer
- [ ] Assign your self
- [ ] Set related Project
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "scripts": {
         "clean": "rimraf dist && npm un -g",
         "compile": "tsc",
+        "lint": "tslint -c tslint.json src/**/*.ts",
```
Comment (pahrizal) [inline package.json:12]: Nice! can you add this lint into the build pipeline also?

File: src/helpers/functions.ts [hunk 2]
```diff
@@ -1,69 +1,69 @@
...
-        .replace(
-            /\$SWAGGER_USER/g,
-            args.swaggerUser || args.appName.toLowerCase()
-        )
-        .replace(
-            /\$SWAGGER_PASS/g,
-            args.swaggerPass || args.appName.toLowerCase()
-        )
-    return stringContents
+  let x: string
```
Comment (pahrizal) [inline src/helpers/functions.ts]: I consider this is not used, is it?

File: package.json (hunk 1 above)
Comment (darmawan01) [reply package.json:12]: sure

File: src/helpers/functions.ts (hunk 2 above)
Comment (darmawan01) [reply src/helpers/functions.ts]: aaa sorry, that is my test var. my bad

**Comment by darmawan01** [comment]
Please review it again

**Review by pahrizal — APPROVED** [review]
Look's good!

**Pull request merged**